fabtests/prov/efa/Makefile.include

```diff
@@ -67,9 +69,10 @@ prov_efa_src_fi_efa_exhaust_mr_reg_rdm_pingpong_SOURCES = \
 	$(benchmarks_srcs)
 prov_efa_src_fi_efa_exhaust_mr_reg_rdm_pingpong_LDADD = libfabtests.la
 
+if BUILD_EFA_RDMA_CHECKER
 prov_efa_src_fi_efa_rdma_checker_SOURCES = \
 	prov/efa/src/efa_rdma_checker.c
 prov_efa_src_fi_efa_rdma_checker_LDADD = libfabtests.la
-prov_efa_src_fi_efa_rdma_checker_LDFLAGS = -lefa
```

Comment:
I would keep this one and avoid changing LIBS.

Reply:
Thx, updated